notebooks/lesson06_model-interpretation.ipynb

```diff
@@ -0,0 +1,2088 @@
+{
```

Comment:
<p>why do you save them? is there are particular reason?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
<p>ah yes this is redundant and left over from some experiments - good catch!</p>